widgets/js/index.tsx

```diff
@@ -66,6 +87,58 @@ function renderChart({ model, el }: RenderArgs) {
   model.on("change:estimates", onChange);
 }
 
+function renderScatterChart({ model, el }: RenderArgs) {
+  const onChange = () => {
+    const results = model.get("estimates");
+
+    const estimates = [];
```

Comment:
Ideally the render call in the widget should just render the component - similar to those above. Does this next 20 lines of logic belong here or in the component? Do we need a new component to wrap it up this logic and the two child components maybe? (Just thinking here - not proposing either of these yet).